### PR TITLE
Update company messaging

### PR DESCRIPTION
**Pull request by apaskulin**
## Description

- Updates high-level company and product descriptions to reflect new messaging.
- Updates product descriptions on docs homepage in light of recent EOL announcements.
- hybrid-cloud --> multi-cloud
- Removes references to Go lang

Proposed product descriptions:

Screen Shot 2019-03-15 at 10 35 03 AM

### CODE REVIEW

File: content/sensu-go/5.0/_index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Sensu Go"
-description: "Sensu is the flexible monitoring event pipeline built to reduce operator burden and meet the challenges of monitoring hybrid-cloud and ephemeral infrastructures. Read the docs to learn more about Sensu."
+description: "Monitor your entire infrastructure, from Kubernetes to bare metal. The Sensu monitoring event pipeline empowers businesses to automate their monitoring workflow and gain deep visibility into their infrastructure, applications, and operations. Read the docs to learn more about Sensu."
```
Comment (bananabele) [inline content/sensu-go/5.0/_index.md]: make "workflows" plural, please

File: content/sensu-go/5.0/_index.md [hunk 2]
```diff
@@ -11,12 +11,10 @@ layout: "single"
...
-Started in 2011, Sensu’s flexible approach solves the challenges of monitoring hybrid-cloud and ephemeral infrastructures with scalable, automated workflows and integrations with tools you already use.
+Sensu is the industry leading solution for multi-cloud monitoring at scale. The Sensu monitoring event pipeline empowers businesses to automate their monitoring workflows and gain deep visibility into their multi-cloud environments. Founded in 2017, Sensu offers a comprehensive monitoring solution for enterprises, providing complete visibility across every system, every protocol, every time — from Kubernetes to bare metal.
 **Get started now and feel the #monitoringlove:** [Learn Sensu Go][1].
 
-Sensu Go is the latest version of Sensu, written in Go and designed to be more portable, easier and faster to deploy, and (even more) friendly to containerized and ephemeral environments.
-But above all, it was designed to support the features and functionality you’ve come to know and love about Sensu.
+Sensu Go is the latest version of Sensu, designed to be more portable, easier and faster to deploy, and (even more) friendly to containerized and ephemeral environments.
```
Comment (bananabele) [inline content/sensu-go/5.0/_index.md:17]: friendly to [monitoring] containerized and ephemeral environments?

File: content/sensu-go/5.1/_index.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Sensu Go"
-description: "Sensu is the flexible monitoring event pipeline built to reduce operator burden and meet the challenges of monitoring hybrid-cloud and ephemeral infrastructures. Read the docs to learn more about Sensu."
+description: "Monitor your entire infrastructure, from Kubernetes to bare metal. The Sensu monitoring event pipeline empowers businesses to automate their monitoring workflow and gain deep visibility into their infrastructure, applications, and operations. Read the docs to learn more about Sensu."
```
Comment (bananabele) [inline content/sensu-go/5.1/_index.md]: same comment about "workflows"

File: content/sensu-go/5.3/_index.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Sensu Go"
-description: "Sensu is the flexible monitoring event pipeline built to reduce operator burden and meet the challenges of monitoring hybrid-cloud and ephemeral infrastructures. Read the docs to learn more about Sensu."
+description: "Monitor your entire infrastructure, from Kubernetes to bare metal. The Sensu monitoring event pipeline empowers businesses to automate their monitoring workflow and gain deep visibility into their infrastructure, applications, and operations. Read the docs to learn more about Sensu."
```
Comment (bananabele) [inline content/sensu-go/5.3/_index.md]: ditto about "workflows"

File: content/sensu-go/5.0/_index.md (hunk 1 above)
Comment (apaskulin) [reply content/sensu-go/5.0/_index.md]: "Monitor your entire infrastructure, from Kubernetes to bare metal. The Sensu monitoring event pipeline empowers businesses to automate their monitoring workflows and gain deep visibility into their infrastructure, applications, and operations. Try Sensu today."

Comment (apaskulin) [inline content/sensu-go/5.0/_index.md]:
```suggestion
description: "Monitor your entire infrastructure, from Kubernetes to bare metal. The Sensu monitoring event pipeline empowers businesses to automate their monitoring workflows and gain deep visibility into their infrastructure, applications, and operations. Read the docs to learn more about Sensu."
```

File: content/sensu-go/5.1/_index.md (hunk 3 above)
Comment (apaskulin) [inline content/sensu-go/5.1/_index.md]:
```suggestion
description: "Monitor your entire infrastructure, from Kubernetes to bare metal. The Sensu monitoring event pipeline empowers businesses to automate their monitoring workflows and gain deep visibility into their infrastructure, applications, and operations. Read the docs to learn more about Sensu."
```

File: content/sensu-go/5.3/_index.md (hunk 4 above)
Comment (apaskulin) [inline content/sensu-go/5.3/_index.md]:
```suggestion
description: "Monitor your entire infrastructure, from Kubernetes to bare metal. The Sensu monitoring event pipeline empowers businesses to automate their monitoring workflows and gain deep visibility into their infrastructure, applications, and operations. Read the docs to learn more about Sensu."
```

File: content/sensu-go/5.2/_index.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Sensu Go"
-description: "Sensu is the flexible monitoring event pipeline built to reduce operator burden and meet the challenges of monitoring hybrid-cloud and ephemeral infrastructures. Read the docs to learn more about Sensu."
+description: "Monitor your entire infrastructure, from Kubernetes to bare metal. The Sensu monitoring event pipeline empowers businesses to automate their monitoring workflow and gain deep visibility into their infrastructure, applications, and operations. Read the docs to learn more about Sensu."
```
Comment (apaskulin) [inline content/sensu-go/5.2/_index.md]:
```suggestion
description: "Monitor your entire infrastructure, from Kubernetes to bare metal. The Sensu monitoring event pipeline empowers businesses to automate their monitoring workflows and gain deep visibility into their infrastructure, applications, and operations. Read the docs to learn more about Sensu."
```

File: content/sensu-go/5.0/_index.md (hunk 2 above)
Comment (cwjohnston) [reply content/sensu-go/5.0/_index.md:17]: In my view it is more friendly both to monitoring such environments and deploying/running in such environments. either either way works for me ¯\_(ツ)_/¯

File: content/sensu-go/5.0/_index.md [hunk 6]
```diff
@@ -11,12 +11,10 @@ layout: "single"
 
 <iframe src="https://ghbtns.com/github-btn.html?user=sensu&repo=sensu-go&type=star&count=true" frameborder="0" scrolling="0" width="87px" height="20px"></iframe> | [Share your feedback][2]
 
-Sensu is the monitoring event pipeline built to reduce operator burden and make developers and business owners happy.
-Started in 2011, Sensu’s flexible approach solves the challenges of monitoring hybrid-cloud and ephemeral infrastructures with scalable, automated workflows and integrations with tools you already use.
+Sensu is the industry leading solution for multi-cloud monitoring at scale. The Sensu monitoring event pipeline empowers businesses to automate their monitoring workflows and gain deep visibility into their multi-cloud environments. Founded in 2017, Sensu offers a comprehensive monitoring solution for enterprises, providing complete visibility across every system, every protocol, every time — from Kubernetes to bare metal.
```
Comment (cwjohnston) [inline content/sensu-go/5.0/_index.md:14]: I'm hesitant to describe Sensu as a comprehensive solution... I feel like this is in conflict with the idea that Sensu is a comprehensive monitoring _framework_ which is used with other tools to build a solution.

**Review by cwjohnston — CHANGES_REQUESTED** [review]

File: content/sensu-go/5.0/_index.md (hunk 6 above)
Comment (apaskulin) [reply content/sensu-go/5.0/_index.md:14]: cc @bananabele

File: content/sensu-go/5.0/_index.md (hunk 2 above)
Comment (apaskulin) [reply content/sensu-go/5.0/_index.md:17]: Thanks for this feedback. I think this is ok to leave as-is.

File: content/sensu-go/5.0/_index.md (hunk 6 above)
Comment (bananabele) [reply content/sensu-go/5.0/_index.md:14]: @cwjohnston @apaskulin that's a great point, though from a corporate and product marketing perspective we are trying to position ourselves as a "solution," namely a "scalable, multi-cloud monitoring solution." We've been moving away from "framework" in our messaging, as it sounds work intensive and therefore off putting to enterprises. Let me know if you have further thoughts -- I'm happy to discuss!

**Review by cwjohnston — APPROVED** [review]

**Pull request merged**